src/main/java/Duke.java

```diff
@@ -1,23 +1,48 @@
+import java.util.ArrayList;
 import java.util.Scanner;
 
 public class Duke {
-    public static void main(String[] args) {
-        String line= "    ____________________________________________________________";
+
+    private static void printIndentedMsg(String msg){
         String indent= "     ";
-        Scanner s = new Scanner(System.in);
-        System.out.println(line);
-        System.out.println(indent + "Hello! I'm Duke");
-        System.out.println(indent + "What can I do for you?");
+        System.out.println(indent + msg);
+    }
+
+    private static void printLine(){
+        String line= "    ____________________________________________________________";
         System.out.println(line);
+    }
+
+    public static void main(String[] args) {
+
+        Scanner s = new Scanner(System.in);
+        printLine();
+        printIndentedMsg("Hello! I'm Duke");
+        printIndentedMsg("What can I do for you?");
+        printLine();
         String msg = s.nextLine();
-        while(!msg.equals("bye")){
-            System.out.println(line);
-            System.out.println(indent + msg);
-            System.out.println(line);
+        ArrayList<String> toDoList = new ArrayList<>();
+        while(!msg.equals("bye")) {
```

Comment:
good approach to terminate the loop. Might be better to use a boolean variable so that you can terminate in other circumstances than the command 'bye' in the future.